src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FileReaderImplTest {
+    private FileReader reader = new FileReaderImpl();
+
+    @Test
+    void read_nullFile_notOk() {
+        String nullName = null;
+        Assertions.assertThrows(RuntimeException.class, () -> reader.read(nullName));
+    }
+
+    @Test
+    void read_notExistingFile_notOk() {
+        String fileName = "notExistingFile.csv";
+        Assertions.assertThrows(RuntimeException.class, () -> reader.read(fileName));
+    }
+
+    @Test
+    void read_correctFile_Ok() {

```

Comment:
The expected list is hard coded. It would be better if you generate it programmatically to make your test more robust and easier to maintain.

Reply:
I'm not sure how this should be corrected...